roles/update/defaults/main.yml

```diff
@@ -17,3 +17,8 @@
 
 # All variables intended for modification should be placed in this file.
 # All variables within this role should have a prefix of "cifmw_update"
+
+cifmw_update_openstack_update_run_target_version: "0.0.2"
+cifmw_update_openstack_update_run_operators_updated: false
+cifmw_update_openstack_update_run_containers_namespace: "podified-antelope-centos9"
+cifmw_update_openstack_update_run_containers_target_tag: "current-podified"
```

Comment:
Please add them to the readme.

Maybe we can add `cifmw_update_openstack_update_run_operators_updated` as well to the readme even if it's a calculated variable that assert if we need to fake the operator update or not.